### PR TITLE
emilk-loguru: try to fix for windows

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

File: subprojects/packagefiles/emilk-loguru/meson_options.txt [hunk 1]
```diff
@@ -104,8 +104,7 @@ option(
 
 option(
   'loguru_stacktraces',
-  type: 'boolean',
-  value: true,
+  type: 'feature',
```
Comment (eli-schwartz) [inline subprojects/packagefiles/emilk-loguru/meson_options.txt:107]: We could add `meson_version: '>=0.47.0'` to the project() declaration too, I guess.

Comment (neheb) [reply subprojects/packagefiles/emilk-loguru/meson_options.txt:107]: feature is already used elsewhere. but sure.

Comment (eli-schwartz) [reply subprojects/packagefiles/emilk-loguru/meson_options.txt:107]: Ah, in that case it *definitely* can't hurt. :D

**Comment by neheb** [comment]
Fixed.

**Pull request merged**